src/amber/cli/templates/app/src/views/layouts/application.slang.ecr

```diff
@@ -15,13 +15,15 @@ html
           == render(partial: "layouts/_nav.slang")
     div.container
       div.row
-        - flash.each do |key, value|
-          div class="alert alert-#{key}"
-            p = flash[key]
+	div.col-sm
```

Comment:
Bad indentation here, when I copy this to my editor the code looks like:

![screenshot_20180411_102512](https://user-images.githubusercontent.com/3067335/38626640-a06adb5c-3d72-11e8-86e4-f12ba26ff77c.png)

Reply:
jquery slim has ajax removed so we probably want the regular build